Sources/CCFilterNode/CCFilterNode.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+private protocol CCFilterNodeDelegate: class {
+    func childNodeDidChangeSelection(_ childNode: CCFilterNode)
+}
+
+public class CCFilterNode {
+
+    // MARK: - Public properties
+
+    public let title: String
+    public let name: String
+    public var value: String?
```

Comment:
Could `value` and `numberOfResults` be constants?

Reply:
The value will change in the range filters, but I think number of results could be constant.

Reply:
So `value` will represent the values retrieved from backend (filter setup) and for some types the current selection for that node? Could be slightly confusing 🤔

Reply:
It represents the same thing, except for most filters the value is static and for others it is dynamic.

Reply:
Hmm, yes I guess that is true. The value to use when filtering - if the node is selected, right?

Reply:
Yep, this is the value we send to the backend if that node is selected ;)